packages/clay-date-picker/src/DayNumber.tsx

```diff
@@ -41,11 +51,28 @@ const ClayDatePickerDayNumber: React.FunctionComponent<IProps> = ({
 				'yyyy MM dd'
 			)}
 			className={classNames}
-			disabled={day.outside}
-			onClick={() => onClick(day.date)}
+			disabled={outside}
+			onClick={() => onClick(date)}
+			onKeyDown={(event) => {
+				// When tabbing and selecting a DayNumber using
+				// SPACE key the active state it's not being removed.
+				// See https://github.com/liferay/clay/issues/3374 for more details.
+				if (event.keyCode === KEY_CODE_SPACE) {
```

Comment:
When I was looking up the keycode stuff, I saw a big red alert on MDN I didn't pay attention to. It looks like `event.keyCode` is gone. They recommend using `event.code` (not supported in IE) and `event.key`. I think the right way going forward is:
```
if (event.key === ' ' || event.key === 'Spacebar') {}
```

Reply:
😱 my +1 for that

Reply:
Thanks for pointing up this Pat. I didn't noticed keyCode isn't recommended more. 

Apparently We're using on Clay in these files:

https://gist.github.com/diegonvs/89654431236967272f4ff3bfc43b8311

Ignoring clay-css/jquery related usages, We're using on the Modal, MultiSelect and on our FocusScope component.

I consider replacing this usages with `.key` the best option but I think We must pay attention in 

> `Has non-standard key identifiers and incorrect behaviour with AltGraph.`

from: https://caniuse.com/#feat=keyboardevent-key

I'll fill an issue for replacing this usages and I'll use `.key` on this PR

Reply:
Yeah, I didn't know that either. This is also used a lot on the Portal. cc @wincent maybe we should create a recommendation in the guidelines to use `key` and `code` instead of `keyCode`?

Reply:
Bringing eslint-unicorn-plugin cool insights, https://github.com/sindresorhus/eslint-plugin-unicorn/blob/master/docs/rules/prefer-event-key.md. Can we enforce this on our eslint config?

Reply:
> Bringing eslint-unicorn-plugin ™️ cool insights, https://github.com/sindresorhus/eslint-plugin-unicorn/blob/master/docs/rules/prefer-event-key.md.

![nogodno](https://user-images.githubusercontent.com/905006/86370168-a860b500-bc7f-11ea-9167-3a6de0ecc6dd.gif)

> Can we enforce this on our eslint config?

Probably yes 😉

Reply:
https://github.com/liferay/clay/issues/3430

Reply:
> Yeah, I didn't know that either. This is also used a lot on the Portal. cc @wincent maybe we should create a recommendation in the guidelines to use `key` and `code` instead of `keyCode`?

Guidelines are cool and all, but I'd much rather have a code abstraction that guides people to use "the right tool". Lint enforcing the use of the tool is a bonus that may be necessary if discoverability of the abstraction isn't good enough. Guidelines are useful support, but let's not kid ourselves that a document in a GitHub repo will stop "Random Product Team X" from shipping something we don't want into "Secret DXP Directory Z".

About the eslint-plugin-unicorn rule, I would steer clear of it. Apart from the undesirability of [the dependency footprint](https://github.com/sindresorhus/eslint-plugin-unicorn/blob/6c02ce4e32c84702bcd06e90aa91696366036294/package.json#L35-L49), I don't think it's going to help us avoid the issues with IE compat noted on #3430 (see [the implementation](https://github.com/sindresorhus/eslint-plugin-unicorn/blob/6c02ce4e32c84702bcd06e90aa91696366036294/rules/prefer-event-key.js#L13-L60), which assumes non-IE browser).

So I'd provisionally say:

![scooby-no](https://user-images.githubusercontent.com/7074/86390795-04840300-bc99-11ea-87b5-55909172deef.gif)

And suggest that if we care about linting, we might even want to make our own.